src/sentry/api/endpoints/organization_events_trends.py

```diff
@@ -256,7 +256,7 @@ def build_result_handler(
         self, request, organization, params, trend_function, selected_columns, orderby, query
     ):
         def on_results(events_results):
-            def get_event_stats(query_columns, query, params, rollup):
+            def get_event_stats(query_columns, query, params, rollup, zerofill_results):
```

Comment:
Can we make sure to pass `zerofill_results` to `top_events_timeseries` here too so it's possible to enable it for trends too?

Reply:
Added to `top_events_timeseries`! Frontend request will need to be updated to make use of this but the support should be there now.